enum.go

```diff
@@ -298,27 +298,6 @@ var AllContactType = []string{
 	string(ContactTypeGitHub),
 }
 
-// TODO: This appears to be duplicative of the above and i'm not sure why we need it
```

Comment:
basically - actioning this TODO - so that we can have enum.go be auto generated at the click of the button